### PR TITLE
Email quotas

### PR DESCRIPTION
**Pull request by TommyUnger**
Adding email "soft quotas"

### CODE REVIEW

File: setup-daily-reports.sh [hunk 1]
```diff
@@ -1,5 +1,13 @@
 QPROFILEPATH=$(ls -1ra ~/.*profile* | tail -1)
 
+python app.py --op verify_config
+RESULT=$?
+if [ $RESULT -ne 0 ] ; then
+    echo "********** config.json verification failed! **********"
```
Comment (FreeAssoc) [inline setup-daily-reports.sh]: :+1:  works!

(daily_storage_reports)➜  daily_storage_reports git:(email_quotas) ./setup-daily-reports.sh
ls: /Users/mmurray/.*profile*: No such file or directory
Verifying config.json
Successfully read and parsed json
Validating Qumulo cluster API connections
Attempting to connect to: <Hostname of Qumulo storage cluster> with <API user name for Qumulo storage cluster> login
********** Qumulo Cluster API connection failure **********
<class 'socket.gaierror'>
********** config.json verification failed! **********
Check your installation, settings and/or fix the config.json file and run this script again.
(daily_storage_reports)➜  daily_storage_reports git:(email_quotas)

File: apitocsv.py [hunk 2]
```diff
@@ -91,7 +91,12 @@ def get_latest_date_dashstats_file(self, csv_file):
     def get_dashstats(self, table_name, time_inteval_seconds=5):
         api_begin_time = int(time.time()-60*60*25) # 24 hours of data, the max Qumulo stores
 
-        res = self.qumulo_api_call(self.api_cli.stats.time_series_get, begin_time=api_begin_time)
+        # handle multiple api versions
+        try:
+            time_series_get_func = self.api_cli.stats.time_series_get
+        except AttributeError:
+            time_series_get_func = self.api_cli.analytics.time_series_get
+        res = self.qumulo_api_call(time_series_get_func, begin_time=api_begin_time)
```
Comment (FreeAssoc) [inline apitocsv.py]: this approach is fine for now, but eventually we'll want to go back and remove the old one.  Maybe add TODO comments to that effect?  Once we sunset 1.2.7/1.2.8...

File: app.py [hunk 3]
```diff
@@ -37,6 +39,56 @@ def get_db(db_name):
     return flask.g.db
 
 
+def nice_bytes(n):
```
Comment (FreeAssoc) [inline app.py]: name's a little funny but OK :)

File: static/dataTables.editor.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
+/*!
```
Comment (FreeAssoc) [inline static/dataTables.editor.js]: does this file need to be checked in?  Do we edit it?

Comment (TommyUnger) [inline static/dataTables.editor.js]: Good question. It's a paid-for version of a library, so yeah. Need to include.

**Comment by FreeAssoc** [comment]
looks good!

**Pull request merged**